packages/optimizer/spec/transformers/SeparateKeyframes/moves_keyframes_with_existing/input.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <script async custom-element=amp-experiment src=https://cdn.ampproject.org/v0/amp-experiment-0.1.js></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <link rel=stylesheet href=https://cdn.ampproject.org/v0.css>
+  <link href=https://fonts.googleapis.com/css?foobar rel=stylesheet type=text/css>
+  <link href=https://example.com/favicon.ico rel=icon>
+  <style amp-custom>
+    .another{display:block}@keyframes anim1{from{opacity:0}top{opacity:1}}.foo{height:0}@media screen and (max-width:960px){@keyframes anim1{from{opacity:1}to{opacity:0}}}.bar{height:auto}
```

Comment:
Can you add another example using `@support` with nested `@keyframe` for completeness sake.

Reply:
Yup, updated